modules/high-sev-security-policies.adoc

```diff
@@ -18,12 +18,13 @@ The following table lists the default security policies in {product-title} that
 [options="header"]
 |====
 |Life cycle stage |Name |Description | Status
-|Build or Deploy |Fixable CVSS >= 6 and Privileged |Alerts when deployments run in privileged mode with fixable vulnerabilities that have a CVSS of at least 6. |Enabled
-|Build or Deploy |Fixable CVSS >= 7 |Alerts when deployments with fixable vulnerabilities have a CVSS of at least 7. |Enabled
+|Build or Deploy |Fixable CVSS >= 7 |Alerts when deployments with fixable vulnerabilities have a CVSS of at least 7. |Disabled
 |Build or Deploy |Fixable Severity at least Important |Alerts when deployments with fixable vulnerabilities have a severity rating of at least Important. |Enabled
 |Build or Deploy |Secure Shell (ssh) Port Exposed in Image |Alerts when deployments expose port 22, which is commonly reserved for SSH access. |Enabled
 |Deploy |Emergency Deployment Annotation |Alerts when deployments use the emergency annotation, such as "admission.stackrox.io/break-glass":"ticket-1234" to circumvent StackRox Admission Controller checks. |Enabled
 |Deploy |Environment Variable Contains Secret |Alerts when deployments have environment variables that contain 'SECRET'. |Enabled
+|Deploy |Fixable CVSS >= 6 and Privileged |Alerts when deployments run in privileged mode with fixable vulnerabilities that have a CVSS of at least 6. |Disabled in version 3.72 and later
+|Deploy | Privileged Containers with Important and Critical Fixable CVEs |Alerts when containers that are running in privileged mode have important or critical fixable vulnerabilities. |Enabled
```

Comment:
Do we want to make a note on the second line here that it's enabled and present in version 3.72 and later?

Reply:
So my thinking was that this chapter is going into 3.72 and later - so we wouldn't need to specify that the policy came in 3.72. (Or that it's not enabled for pre-3.72 releases.)

Reply:
Good point, @kcarmichael08! I will update that line to remove the mention of 3.72. Thanks!